src/lib.rs

```diff
@@ -375,7 +375,7 @@ impl<Fs: FileSystem> ResolverGeneric<Fs> {
         ctx: &mut Ctx,
     ) -> Result<CachedPath, ResolveError> {
         // Make sure only relative or normal paths gets called
-        debug_assert!(Path::new(specifier).components().next().is_some_and(|c| matches!(
+        debug_assert!(Path::new(specifier).components().next().map_or(true, |c| matches!(
```

Comment:
I used [`map_or`](https://doc.rust-lang.org/std/option/enum.Option.html#method.map_or) instead of [`is_none_or`](https://doc.rust-lang.org/std/option/enum.Option.html#method.is_none_or) as it required bumping MSRV.

Reply:
This debug assertion seems to be working as intended, let me double check.

Reply:
Ah, sorry I misunderstood the code here. I'll update with the correct fix.

Reply:
Pushed a commit with the correct fix 👍